odd-platform-api/src/main/java/com/provectus/oddplatform/repository/DatasetFieldRepositoryImpl.java

```diff
@@ -166,4 +176,85 @@ public void updateSearchVectors(final long datasetFieldId) {
             Map.of(labelName, LABEL.NAME)
         ).execute();
     }
+
+    @Override
+    @Transactional
+    public DatasetFieldDto updateDatasetField(final long datasetFieldId,
+                                              final DatasetFieldUpdateFormData datasetFieldUpdateFormData) {
+        setDescription(datasetFieldId, datasetFieldUpdateFormData.getDescription());
+
+        final Set<String> names = new HashSet<>(datasetFieldUpdateFormData.getLabelNames());
```

Comment:
The same here. If we want to update only description labels can be null or empty list. I think we need to do additional check here